romanisim/tests/test_persistence.py

```diff
@@ -0,0 +1,51 @@
+"""Unit tests for persistence module."""
+
+import numpy as np
+from romanisim import persistence
+from romanisim import parameters
+
+
+def test_fermi():
+    # fermi(x, dt, A, x0, dx, alpha, gamma)
+    answers = [(1, 1000, 1, 1, 1, 1, 1, 0.5),
+               (100, 1000, 1, 1, 1, 1, 1, 100),
+               (100, 1000, 2, 1, 1, 1, 1, 200),
+               (100, 1000, 1, 1, 1, 2, 1, 10000),
+               (100, 1000, 1, 100, 1, 1, 1, 0.5),
+               (100, 1000, 1, 1, 1, 1, 2, 100),
+               (100, 2000, 1, 1, 1, 1, 1, 50),
+               (100, 2000, 1, 1, 1, 1, 2, 25)]
+    hw = parameters.persistence['half_well']
+    parameters.persistence['half_well'] = 0.1
```

Comment:
If I understood the test correctly, you never check the case where x is less than the half-well level. If so, I would recommend to add one more line where you check that one.

Reply:
I've added some more here, thanks.

Reply:
Thanks! Looks good.